lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -32,6 +32,13 @@
   min-width: $nx-page-width-min;
 }
 
+.nx-page--sidebar {
+  @include container-horizontal;
+
+  align-items: flex-start;
```

Comment:
Probably want `align-items` stretch.  I'd expect both the sidebar and the `.nx-page-content` to be full height at all times. As it is now the `.nx-page-content` is only as tall as its contents.  This would also allow you to get rid of the `vh` height on the sidebar.

Reply:
> Probably want `align-items: stretch`. I'd expect both the sidebar and the `.nx-page-content` to be full height at all times. As it is now the `.nx-page-content` is only as tall as its contents. This would also allow you to get rid of the `vh` height on the sidebar.

No longer relevant as of b9b047801218fedc6d13234b0795b2c66b9c2238